protobuf-kafkaconnect-converter/src/test/proto/ArrayTypeSyntax3.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package com.amazonaws.services.schemaregistry.kafkaconnect.tests.syntax3;
+
+message ArrayType {
+  repeated string str = 1;
+  repeated int32 i32 = 2;
+  repeated bool boolean = 3;
```

Comment:
Is it possible to have repeated maps?

Reply:
Maps cannot be repeated, optional, or required according to the documentation. But repeated can be applied to message. Since we haven't implemented message conversion, we can test it in the future.